auctions/english-auction/src/main.sw

```diff
@@ -0,0 +1,377 @@
+contract;
+
+dep abi;
+dep data_structures;
+dep errors;
+dep events;
+dep utils;
+
+use abi::{EnglishAuction, NFT};
+use data_structures::*;
```

Comment:
Explicit imports should work

Reply:
Last I checked it didn't work with my trait but I'll try again.

Reply:
Update toolchain, it has been fixed in 0.16.2 iirc